src/aks-preview/azext_aks_preview/tests/latest/test_aks_commands.py

```diff
@@ -165,7 +171,7 @@ def test_aks_create_with_ingress_appgw_addon(self, resource_group, resource_grou
             'name': aks_name
         })
 
-        create_cmd = 'aks create --resource-group={resource_group} --name={name} --enable-managed-identity --service-principal xxxx --client-secret yyyy --generate-ssh-keys ' \
+        create_cmd = 'aks create --resource-group={resource_group} --name={name} --enable-managed-identity --generate-ssh-keys ' \
```

Comment:
why the `--service-principal` and `--client-secret` be removed? this command will fail when running live test